src/dymaptic.GeoBlazor.Core/Components/Layers/Layer.cs

```diff
@@ -264,14 +264,26 @@ public override async ValueTask DisposeAsync()
     ///     A cancellation token to cancel the operation.
     /// </param>
     public async Task Load(IJSRuntime jsRuntime, JsModuleManager jsModuleManager,
-        CancellationToken cancellationToken = default)
+        CancellationToken cancellationToken)
     {
         JsRuntime ??= jsRuntime;
         JsModuleManager ??= jsModuleManager;
 
         await Load(cancellationToken);
     }
 
+    /// <summary>
+    ///     Loads the resources referenced by this class. This method automatically executes for a View and all of the
+    ///     resources it references in Map if the view is constructed with a map instance.
+    ///     This method must be called by the developer when accessing a resource that will not be loaded in a View.
+    ///     The load() method only triggers the loading of the resource the first time it is called. The subsequent calls
+    ///     return the same promise.
+    /// </summary>
+    public Task Load()
+    {
+        return Load(CancellationToken.None);
+    }
```

Comment:
This type of duplication helps with the Roslyn PublicApiAnalyzer we used to generate release notes.